tests/blob/blob_module/Makefile

```diff
@@ -0,0 +1,2 @@
+BLOBS += blobtest.bin
```

Comment:
This file is missing ;)

Reply:
thanks! It was blanked out by .gitignore...